packages/components/src/components/tab-header/tab-header.css

```diff
@@ -78,6 +79,7 @@
   position: absolute;
   height: var(--tab-border-size);
   background-color: var(--tab-border-color);
+  z-index: -1;
```

Comment:
@marvinLaubenstein this is what's causing the issue in Storybook (honestly don't know why, since outside Storybook it seems to work) — but in any case, is this `z-index: -1` really needed?

Reply:
Thanks, I am already desperate 🙏🏻